docs/installation.rst

```diff
@@ -0,0 +1,6 @@
+Installation
```

Comment:
This a separate point but should we have develop install instructions on the github somewhere?

Reply:
Yes, I guess they can be here too, and the readme should point to this

Reply:
I added to the documentation.